resources/js/views/Teams/CreateTeam.vue

```diff
@@ -0,0 +1,188 @@
+                    <p>Anyone with this ID will be able to join your team.</p>
+                    <span
+                        class="is-danger"
+                        v-if="errorExists('identifier')"
+                        v-text="getFirstError('identifier')"
+                    />
+                    <input
+                        class="input mb2"
+                        name="identifier"
+                        placeholder="Awesome2020"
+                        required
+                        v-model="identifier"
+                        @keydown="clearError('identifier')"
+                    />
+
+                    <!-- Todo - Checkbox -->
+                    <!-- Allow people to join your team automatically? -->
+                    <!-- Yes = auto-join is on -->
+                    <!-- No = approval is required -->
+
+                    <div>
+                        <button class="button is-medium" @click.prevent="goback">Cancel</button>
+                        <button :class="button" :disabled="processing">Create Team</button>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+/* Todo - translations */
+export default {
+    name: 'CreateTeam',
+    data ()
+    {
+        return {
+            btn: 'button is-medium is-primary',
+            processing: false,
+            identifier: '',
+            name: '',
+            teamType: 1
+        };
+    },
+    computed: {
+
+        /**
+         * Add spinner when processing
+         */
+        button ()
+        {
+            return this.processing ? this.btn + ' is-loading' : this.btn;
+        },
+
+        /**
+         * Errors object from teams
+         */
+        errors ()
+        {
+            return this.$store.state.teams.errors;
+        },
+
+        /**
+         * Number of teams the user is allowed to create
+         */
+        remaining ()
+        {
+            return this.user.remaining_teams;
+        },
+
+        /**
+         * Types of teams from the database
+         */
+        teamTypes ()
+        {
+            return this.$store.state.teams.types;
+        },
+
+        /**
+         * Currently authenticated user
+         */
+        user ()
+        {
+            return this.$store.state.user.user;
+        }
+    },
+    methods: {
+
+        /**
+         * Clear an error with this key
+         */
+        clearError (key)
+        {
+            if (this.errors[key]) this.$store.commit('clearTeamsError', key);
+        },
+
+        /**
+         * Create a new team
+         */
+        async create ()
+        {
+            this.processing = true;
+
+            await this.$store.dispatch('CREATE_NEW_TEAM', {
+                name: this.name,
+                identifier: this.identifier,
+                teamType: this.teamType
+            });
+
+            this.processing = false;
```

Comment:
This line will execute when creating a new team fails?

Reply:
Yes, processing will stop (spinner + disabled). But dynamic error object will show to the user